packages/financial-templates-lib/README.md

```diff
@@ -0,0 +1,83 @@
+# @uma/financial-templates-lib
+
+This package contains various clients and helpers to interact with UMA's financial templates. It is primarily used to
+power the disputer, liquidator, and monitor bots.
+
+## Installing the package
+
+```bash
+yarn add @uma/financial-templates-lib
+```
+
+## Importing the package
+
+```js
+const { ExpiringMultiPartyClient, GasEstimator } = require("@uma/financial-templates-lib")
+```
+
+## Clients
+
+The three clients available are:
+
+1. The `ExpiringMultiPartyClient` can be used to access information about EMP sponsors and their collateralization
+   ratios. To understand how to interact with the client, see the class documentation
+   [here](./src/clients/ExpiringMultiPartyClient.js).
+
+1. The `ExpiringMultiPartyEventClient` can be used to access historical events that were emitted by an EMP. To
+   understand how to interact with the client, see the class documentation
+   [here](./src/clients/ExpiringMultiPartyEventClient.js).
+
+1. The `TokenBalanceClient` tracks the collateral and synthetic balances for a list of wallets. To understand how to
+   interact with the client, see the class documentation
+   [here](./src/clients/TokenBalanceClient.js).
+
+## Price Feeds
+
+The package offers a variety of price feed implementations that adere to the `PriceFeedInterface` (docs can be found in
+the interface file [here](./src/price-feed/PriceFeedInterface.js)):
+
+- The `CryptoWatchPriceFeed`, found [here](./src/price-feed/CryptoWatchPriceFeed.js), uses https://cryptowat.ch/ as a
```

Comment:
Nice section here, I will add the `BasketSpreadPriceFeed` for the stablespread-esque products in my PR